tf/env/production/cluster.tf

```diff
@@ -59,3 +59,13 @@ resource "google_container_node_pool" "wbaas-3_highmem-16" {
         max_unavailable = 0
     }
 }
+
+resource "kubernetes_config_map" "cluster" {
```

Comment:
I believe that only sticking this in the default ns is fine since it should only be needed by web pods not adhoc/api jobs etc.